compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.td

```diff
@@ -142,7 +144,9 @@ def AMDAIETileAndFuse :
     Option<"useSCFFor", "use-scf-for", "bool", /*default=*/"false",
       "Use scf.forall by default for the corresponding tiling level">,
     Option<"tilingLevel", "tiling-level", "int64_t", /*default=*/"-1",
-      "Use default tiling level used to retrieve the configuration from lowering_config">  
+      "Use default tiling level used to retrieve the configuration from lowering_config">,
+    Option<"targetOp", "target-op", "int64_t", /*default=*/"1",
+      "Target consumer op for tiling, 1 for linalg.matmul, 2 for weight linalg.copy and 3 for input linalg.copy">
```

Comment:
Better to create an Enum or an equivalent to make this verbose.

Then even the API `consumerToSkip` (which uses this) would be more "readable".

Reply:
I tried to use llvm;:cl::values(clEnumValN(1, ...), clEnumValN(2, ...), clEnumValN(3, ...)) for the options, but got build errors. Let me know if you have a better working solution for this.

Reply:
I'd have based it on @jtuyls 's PR : https://github.com/nod-ai/iree-amd-aie/pull/121 + defining the Enums too.

Perhaps @jtuyls can also help if we know what build errors you're facing.

Reply:
I've figured this out. Now it's more readable.